sdk/src/types.ts

```diff
@@ -117,6 +126,7 @@ export type DepositRecord = {
 	bankIndex: BN;
 	amount: BN;
 	oraclePrice: BN;
+	referrer: PublicKey;
```

Comment:
wouldn't we always know a userAuthority is tied to a single referrer anyways?

Reply:
i did this to make off chain indexing queries easier

Reply:
think we should either store referrer in event logs or just store cumulative referrer stats on user stats